src/components/Input/Input.tsx

```diff
@@ -143,6 +145,18 @@ const StyledInput: any = styled.input<IStyledInput>`
   ::placeholder {
     color: ${({ theme: { knitui } }) => knitui.inputPlaceholderColor};
   }
+  &:disabled {
+    cursor: not-allowed;
```

Comment:
we can assign all changes required when element is disabled in this selector instead.